sass/_FacetValues.scss

```diff
@@ -179,6 +182,7 @@
   white-space: nowrap;
   text-overflow: ellipsis;
   vertical-align: middle;
+  flex-grow: 1;
```

Comment:
Possible alternatives would be:
* Replacing this with `margin-right: auto;`
* Adding `margin-left: auto;` to the count.